### ISSUE TITLE
[Slider] Wrong slider-fill-width

### ISSUE THREAD

**Issue by XmlmXmlmX** [zurb/foundation-sites#9408]
**How can we reproduce this bug?**

Create a [Two Handles Slider](http://foundation.zurb.com/sites/docs/slider.html#two-handles) like this example (```data-start``` and ```data-initial-start``` to ```1```,  ```data-end``` and ```data-initial-end``` to ```5```):

```
<div class="stars slider" data-slider data-start="1" data-end="5" data-step="1" data-initial-start="1" data-initial-end="5">
    <span class="slider-handle" data-slider-handle role="slider" tabindex="1" aria-controls="ratingFrom"></span>
    <span class="slider-fill" data-slider-fill></span>
    <span class="slider-handle" data-slider-handle role="slider" tabindex="1" aria-controls="ratingTo"></span>
</div>
```

**What did you expect to happen?**

the ```.slider-fill```-element should be span between both handles.

**What happened instead?**

the ```.slider-fill```-element is shorter than expected (initially). But when dragging one of the handles, the filling is getting it's correct width.

**Test case:**

JSFiddle: https://jsfiddle.net/Mescaline/dw1jcc9j/

**Comment by kball** [comment]
This has been fixed in version 6.3 (I just confirmed using your testcase.)  The release candidate for 6.3 will be released this Thursday (12/1) with the final release scheduled for 12/15.

**Comment by XmlmXmlmX** [comment]
Fine, thx.
